config/application.rb

```diff
@@ -23,5 +24,6 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     config.active_record.schema_format = :sql
+    config.time_zone = "Europe/Bratislava"
```

Comment:
Pri registri adries chybaju casove zony a Rails pouzije pri `Time.parse` casovu zonu systemu na ktorom bezi. Tomu som sa chcel vyhnut, ale suhlasim ze mozno to nemalo ist az do globalneho configu. Na druhej strane, vacsina dat v harvestri bude zrejme specificka pre Slovensko, takze aj lokalna casova zona mi dava zmysel.

FWIW, data sa stale ukladaju v UTC. Ak ich chceme aj prezentovat v UTC, mozem ten config zrusit a byt explicitny len pri parsovani registra adries. Chces to radsej defaultne v UTC?